config/schema/oe_authentication.schema.yml

```diff
@@ -16,4 +16,7 @@ oe_authentication.settings:
       label: 'Application assurance levels'
     ticket_types:
       type: string
-      label: 'Application available ticket types'
\ No newline at end of file
+      label: 'Application available ticket types'
+    redirect_user_register_route:
```

Comment:
The `cas` module already offers a similar setting called `prevent_normal_login`. Maybe for consistency we could have the same as `prevent_normal_register`.
Also, that would be a good contribution to cas itself instead of forcing the redirection in the `oe_authentication` module but it seems that they are not very eager to do so since there are responses like this https://www.drupal.org/project/cas/issues/2884216

Reply:
This is "case close" according to https://www.drupal.org/project/cas/issues/2884216. The redirect is provided by `oe_authentication`, not `cas`. So, we have to fix the problem here, not in `cas`.